src/omv/framebuffer.c

```diff
@@ -0,0 +1,75 @@
+/*
+ * This file is part of the OpenMV project.
+ * Copyright (c) 2013/2014 Ibrahim Abdelkader <i.abdalkader@gmail.com>
+ * This work is licensed under the MIT license, see the file LICENSE for details.
+ *
+ * Framebuffer stuff.
+ *
+ */
+#include "imlib.h"
+#include "omv_boardconfig.h"
+#include "framebuffer.h"
+// If buffer size is bigger than this threshold, the quality is reduced.
+// This is only used for JPEG images sent to the IDE not normal compression.
+#define JPEG_QUALITY_THRESH     (160*120*2)
+
+extern char _fb_base;
+framebuffer_t *fb = (framebuffer_t *) &_fb_base;
```

Comment:
Yeah I know how the current method works (I wrote it) and I know it takes space (but Not a lot of data, 8 bytes per include), and I wanted to move those to .c but this contaminates the namespace (can't have another variable called fb for example). Just saying consider either prefixing or using getters.
I'd prefer getters, because this allows us in the future to add code (if needed) before getting the FB.
Re code that will break, it should be refactored first, since we're refactoring anyway let's just fix it, I can do that if you want.